lib/kitchen/driver/oci.rb

```diff
@@ -100,23 +83,51 @@ def destroy(state)
         return unless state[:server_id]
 
         instance.transport.connection(state).close
-        comp_api.terminate_instance(state[:server_id])
+
+        if instance_type == 'compute'
+          comp_api.terminate_instance(state[:server_id])
+        elsif instance_type == 'dbaas'
+          dbaas_api.terminate_db_system(state[:server_id])
+        end
```

Comment:
We ought to add a check in here to raise an exception if the instance_type parameter is set to any other value, so that the user doesn't get a silent failure if they have a typo and then wonder what's going on.